.github/workflows/dashboard-done.yml

```diff
@@ -0,0 +1,33 @@
+on:
```

Comment:
A couple of questions:

1. Will this workflow launch for a pull-request to _any_ branch on this repo?
2. Would it be possible to add an SPDX license header?
```
# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Reply:
Updated in #1896